api/src/main/resources/api/service_openapi.yaml

```diff
@@ -20,8 +20,166 @@ paths:
         500:
           description: Service is broken
 
+  # TODO implement "real" export endpoints
+  '/underlays/{underlay}/entities/{entity}/filters:generateSqlQuery':
```

Comment:
Will "filters" be a real noun in the API, aside from being a model that's passed around in requests? If not - then probably unnecessary in the URL

Reply:
So far, I'm thinking of keeping it as a replacement for 'cohorts' and 'concept sets.' but if not, then some other noun.

Reply:
I see - in this case, the `entities/{entity_id}` infix is probably unnecessary. While the primary entity is an attribute and key trait of a cohort / filter, it is not really a "parent" in my view - it also will typically reference many other entities.